src/Blocks/BlockAnvil.h

```diff
@@ -2,7 +2,7 @@
 #pragma once
 
 #include "BlockHandler.h"
-#include "Mixins.h"
+#include "Mixins/Mixins.h"
 #include "../Entities/Player.h"
 #include "../UI/AnvilWindow.h"
```

Comment:
I haven't changed it in this PR: just moved the Mixins header into its own folder, in preparation for splitting it in the future - the new mixins required their own file to avoid circular dependency iessus.

I can't remember that we made the change (probably just forgotten!) - can you link the decision and then we can put it in the CONTRIBUTING.md file.

Reply:
Oh, it is in the CONTRIBUTING file, I just forgot it :P

Not something for this PR to change, but noted for the future :)